lib/identity/account.rb

```diff
@@ -109,17 +109,23 @@ class Account < Sinatra::Base
         begin
           # confirming an e-mail change requires authentication
           raise Identity::Errors::LoginRequired if !@cookie.access_token
-          api = HerokuAPI.new(user: nil, pass: @cookie.access_token,
-            ip: request.ip, request_ids: request_ids, version: 2)
-          # currently returns a 302, but will return a 200
-          api.post(path: "/confirm_change_email/#{token}", expects: [200, 302])
+          api = HerokuAPI.new(
+            user:        nil,
+            pass:        @cookie.access_token,
+            ip:          request.ip,
+            request_ids: request_ids,
+            version:     3)
+          api.patch(
+            path:    "/users/~",
```

Comment:
Unnecessary spacing detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.